eng/pipelines/MIDebugEngine-CI.yml

```diff
@@ -2,7 +2,7 @@
 name: $(Date:yyyMMdd).$(Rev:r)
 stages:
 - stage: CodeAnalysis
-  dependsOn: []
+  dependsOn: [CI]
```

Comment:
nit: move the stages around so the `stage` at the very top has no dependencies. Makes it easier to read this file if you start with the stage that would start first.